src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -185,6 +185,30 @@ funcs_metadata_output:close()
 -- start building the dispatch wrapper output
 local output = io.open(dispatch_outputf, 'wb')
 
+
+-- ===========================================================================
+-- NEW API FILES MUST GO HERE.
+--
+--  When creating a new API file, you must include it here,
+--  so that the dispatcher can find the C functions that you are creating!
+-- ===========================================================================
+output:write([[
```

Comment:
Some non-API headers that were removed from `api/private/dispatch.c` may also need to be moved here. For example `logmsg()` is generated by this file.